app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/HowCanWeIdentifyYouController.scala

```diff
@@ -68,7 +68,6 @@ class HowCanWeIdentifyYouController @Inject() (
               Redirect(GYEHowCanWeIdentifyYouNinoController.form(service))
             case Some("utr") =>
               Redirect(GYEHowCanWeIdentifyYouUtrController.form(service))
-            case _ => throw new IllegalArgumentException("Required formData to include nino or utr selection")
```

Comment:
Lets not remove this, it's at least a more meaningful exception than the otherwise.